File: src/main/java/p00/Problem_0028.java
package p00;

public class Problem_0028 {

	public static int solve() {

		int solution = 1;
		int number = 1;
		int add = 2;

		for (int i = 0; i < 500; i++) {
			solution += number * 4 + add * 10;
			number += add * 4;
			add += 2;
		}
		return solution;
	}

}

File: src/main/kotlin/p00/Problem0028.kt
package p00

class Problem0028 {
    fun solve(): Int {
        var solution = 1
        var number = 1
        var add = 2
        for (i in 0..499) {
            solution += number * 4 + add * 10
            number += add * 4
            add += 2
        }
        return solution
    }
}